ResearchKit/Common/ORKScaleSliderView.m

```diff
@@ -67,8 +72,31 @@ - (instancetype)initWithFormatProvider:(id<ORKScaleAnswerFormatProvider>)formatP
     if (self) {
         _formatProvider = formatProvider;
         
-        self.leftRangeLabel.text = [formatProvider localizedStringForNumber:[formatProvider minimumNumber]];
-        self.rightRangeLabel.text = [formatProvider localizedStringForNumber:[formatProvider maximumNumber]];
+        UIView *rightRangeView = nil;
+        UIView *leftRangeView = nil;
+        
+        if ([[formatProvider minimumImage] isKindOfClass:[UIImage class]]) {
```

Comment:
Yeah, what you say makes sense. I think that the intent is clearer when you just check for `nil`. ;-)